R/shuffle_ard.R

```diff
@@ -223,8 +224,119 @@ shuffle_ard <- function(x, trim = TRUE) {
   }
 }
 
+#' Rename ARD Columns
+#'
+#' This function combines a pair of `group`/`group_level` or `variable`/`variable_level` columns into a
+#' single column. The `group_level` or `variable_level` column is renamed according to the value of
+#' the `group` or `variable` column, respectively.
+#'
+#' @param x (`data.frame`)\cr
+#'   a data frame
+#' @param col ([`tidy-select`][dplyr::dplyr_tidy_select])\cr
+#'  Name of column containing the variable names
```

Comment:
In the other roxygen comments, we indent this line 2 spaces from the line above. (Sorry, this is an annoying comment)